data/maps/objects/ChampionsRoom.asm

```diff
@@ -10,7 +10,7 @@ ChampionsRoom_Object:
 	def_bg_events
 
 	def_object_events
-	object_event  4,  2, SPRITE_BLUE, STAY, DOWN, 1 ; person
-	object_event  3,  7, SPRITE_OAK, STAY, UP, 2 ; person
+	object_event  4,  2, SPRITE_BLUE, STAY, DOWN, TEXT_CHAMPIONSROOM_RIVAL_INTRO_AND_AFTER_BATTLE ; person
+	object_event  3,  7, SPRITE_OAK, STAY, UP, TEXT_CHAMPIONSROOM_OAK_SAYS_PLAYERS_NAME ; person
```

Comment:
Because of the strong association between text IDs and NPC objects, I think more concise names like `TEXT_CHAMPIONSROOM_RIVAL` and `TEXT_CHAMPIONSROOM_OAK` are appropriate.

For text IDs not associated with an object (usually text IDs used in assembly code in the script file), then having lengthier names to capture context is good.